common.lic

```diff
@@ -349,6 +349,23 @@ module DRC
     def short_regex
       @adjective ? /\b#{adjective}.*\b#{@name}/i : /\b#{@name}/i
     end
+
+    # Convenience method to parse the text of an item as shown when in your hands, with or without an adjective,
+    # into an Item class instance. Originally designed to support DRCI and equipmanager methods.
+    def self.from_text(text)
```

Comment:
I guess I see why you're doing this, but it's a pretty weakly built item (missing 95% of its properties, etc.).

Reply:
Yeah, the goal is to take the text of an item, get it into the Item class in a consistent way **then** make use of the Item class' `short_regex` and `short_name` properties.

In short, have common-items and equipmanager standardize onto a canonical internal data model for items and the regex pattern to match them.

Outside of equipmanager's use, yeah, the additional properties just aren't being used.

We _could_ in good object-oriented or composition design have a simpler class that's for the item adj/name and then the class we have today would extend or use that class.